config/manager/controller_manager_config.properties

```diff
@@ -3,4 +3,5 @@ METRICS_ADDR=:8080
 PROBE_ADDR=:8081
 LEADER_ELECT=false
 WEBHOOK_PORT=9443
-OBC_AUTO_CREATE=true
\ No newline at end of file
+OBC_AUTO_CREATE=true
+EDGEDEPLOYMENT_CONCURRENCY=5
```

Comment:
I'd avoid exposing it, there is a default of 5 internally and exposing it might give the user the impression that this can be tweaked as needed.

Reply:
Piotr asked for it. So I think you 2 need to talk about it.
My personal opinion is that it makes no difference.
No one changes things without checking what it means. Especially if they don't know what it is.
If I remove this line and the configMap won't have this field the code will still work, correct?
So actually, we're talking about this file only, correct?

Reply:
yes, it will work, the value will be taken from its default in code.